### ISSUE TITLE
TODO

### ISSUE THREAD

**Issue by ImagicalGamer** [ImagicalGamer/EconomyPlus-Shop#1]
Shop Signs = 0% Complete
Buy Method = 0% Complete
Sellhand = 0% Complete

**Comment by ImagicalGamer** [comment]
@remotevase @Taki21 @TheDragonRing What if we had an item file that the user could select prices for items and we could have /sellhand?

**Comment by TheDragonRing** [comment]
Good idea! Btw one thing I can't remember if you've added in or not -> Have you added something in that checks your money balance to make sure you aren't buying something for $5000 when you only have $3
